packages/ngworker/lumberjack/src/lib/time/lumberjack-time.service.spec.ts

```diff
@@ -4,6 +4,9 @@ import { LumberjackTimeService } from './lumberjack-time.service';
 
 describe(LumberjackTimeService.name, () => {
   beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [LumberjackTimeService],
```

Comment:
suggestion: Use `LumberjackModule.forRoot` instead of providing each dependency individually
```suggestion
      imports: [LumberjackModule.forRoot()],
```
(also add a relative import at the top of the file)